htdocs/mrp/class/mo.class.php

```diff
@@ -1124,7 +1124,20 @@ public function getNomUrl($withpicto = 0, $option = '', $notooltip = 0, $morecss
 				$label = $langs->trans("ShowMo");
 				$linkclose .= ' alt="'.dol_escape_htmltag($label, 1).'"';
 			}
-			$linkclose .= ' title="'.dol_escape_htmltag($label, 1).'"';
+			$labelProd = '';
+			$subProd = new Product($this->db);
+			$resSub = $subProd->fetch($this->fk_product);
```

Comment:
Making a fetch of data inside a method whose goal is to output data is not a good pratice.
I recommand to add a property $this->product_ref and to have it set during the fetch of $this by the caller.